model.py

```diff
@@ -3389,14 +3278,11 @@ def from_identifiers(cls, _db, identifiers, base_query=None):
             base_query = _db.query(Work).join(Work.license_pools).\
                 join(LicensePool.identifier)
 
-        equivalent_identifier = aliased(Identifier)
-        query = base_query.outerjoin(Identifier.equivalencies).\
-                outerjoin(equivalent_identifier, Equivalency.output).\
-                filter(or_(
-                    Identifier.id.in_(identifier_ids),
-                    and_(equivalent_identifier.id.in_(identifier_ids),
-                    Equivalency.strength>=1)
-                ))
+        identifier_ids_subquery = Identifier.recursively_equivalent_identifier_ids_query(
+            Identifier.id, levels=1, threshold=0.999)
```

Comment:
OK, > 0.999 is fine. I was worried it was >= 0.999 and we would miss identities that we're absolutely sure about, like Overdrive ID -> ISBN.